db/builder.cc

```diff
@@ -57,7 +57,7 @@ Status BuildTable(
     const std::string& dbname, VersionSet* versions,
     const ImmutableDBOptions& db_options, const TableBuilderOptions& tboptions,
     const FileOptions& file_options, TableCache* table_cache,
-    InternalIterator* iter,
+    TableReaderCaller table_reader_caller, InternalIterator* iter,
```

Comment:
Isn't this always `kFlush`? I think compaction uses the lower-layer function in this file (`NewTableBuilder()`).

Reply:
Repair also calls this